sdk/spring/azure-spring-cloud-stream-binder-servicebus-queue/README.md

```diff
@@ -126,7 +126,8 @@ SessionID | com.azure.spring.integration.servicebus.converter.ServiceBusMessageH
 CorrelationId | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.CORRELATION_ID | String | N/A
 To | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.TO | String | N/A
 ReplyToSessionId | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.REPLY_TO_SESSION_ID | String | N/A
-PartitionKey | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.PARTITION_KEY | String | N/A
+**PartitionKey** | com.azure.spring.integration.servicebus.converter.ServiceBusMessageHeaders.PARTITION_KEY | String | 1
+**PartitionKey** | com.azure.spring.integration.core.AzureHeaders.PARTITION_KEY | String | 2
```

Comment:
Could you add some description for the precedence of different headers/properties here cause its a bit complex?